languages.toml

```diff
@@ -1036,6 +1036,8 @@ file-types = ["tex", "sty", "cls", "Rd", "bbx", "cbx"]
 comment-token = "%"
 language-servers = [ "texlab" ]
 indent = { tab-width = 4, unit = "\t" }
+formatter = { command = "latexindent", args = [ "-m" ] }
+auto-format = true
```

Comment:
```suggestion
```

we generally only enable `auto-format` by default if the language is formatted universally. I don't think latex documents are formatted very often so let's leave it off by default

Reply:
I realised that formatting $\LaTeX$ with latexindent is already a [default feature of texlab](https://github.com/latex-lsp/texlab/wiki/Configuration#texlablatexformatter). It requires some tweaking to make it work just like the Latex Workshop plugin from VSCode, which I am more familiar with. Furthermore, they have plans to make their own $\LaTeX$ formatter, which is not a scrappy Perl script and replace the old one.

I can push suggested changes but I'm not sure if this PR is necessary anymore.

Btw I really appreciate your contribution to Helix and can't wait for the public release of Spellbook.